### PR TITLE
mqbblp::Routers::AppContext: Fix function documentation

### PR DESCRIPTION
**Pull request by kaikulimu**

### CODE REVIEW

File: src/groups/mqb/mqbblp/mqbblp_routers.h [hunk 1]
```diff
@@ -758,8 +758,10 @@ class Routers {
         void apply();
 
         // ACCESSORS
-        void loadInternals(mqbcmd::RoundRobinRouter* out) const;
+
         // Load into the specified 'out', internal details about this object.
```
Comment (pniedzielski) [inline src/groups/mqb/mqbblp/mqbblp_routers.h:762]: Requesting changes on a three line diff 😢 

`//` -> `///`

**Review by pniedzielski — CHANGES_REQUESTED** [review]

File: src/groups/mqb/mqbblp/mqbblp_routers.h (hunk 1 above)
Comment (kaikulimu) [reply src/groups/mqb/mqbblp/mqbblp_routers.h]: d

**Comment by kaikulimu** [comment]
Fixed feedback

**Review by pniedzielski — APPROVED** [review]
I may have to approve again once I have write access back...

**Pull request merged**